### DISCUSSION TITLE
merge Initial test and retest

### DISCUSSION THREAD

**Discussion by Dkryptur** [DefectDojo/django-DefectDojo#10754] [Q&A]
Hi Team, How can I merge initial test findings and retest ?
After creating 2 different engagements or tests, vulnerabilities are getting doubled.

**Comment by mtesauro** [comment]
These kind of usage questions are best asked on OWASP's Slack in the #DefectDojo channel see https://github.com/DefectDojo/django-DefectDojo?tab=readme-ov-file#community-getting-involved-and-updates

You can also look at dedup and reimport in the docs:
* https://documentation.defectdojo.com/usage/features/
* https://documentation.defectdojo.com/integrations/importing/
